### PR TITLE
Updated user profile for Ryan Stewart.

### PR DESCRIPTION
**Pull request by ryanstwrt**
Alright, I am a bit rusty, but this should be all clean to be merged with the master copy.

### CODE REVIEW

**Comment by nickwhitman1993** [comment]
@rtrp/admins 
@ryanstwrt I found a bunch of extra white space in your publication section

File: users/stewryan/index.md [hunk 1]
```diff
@@ -8,23 +8,27 @@ author: Ryan Stewart
...
 
 ### Research Interests
 1. Fast Reactor Core Analysis
 2. Reactor Physics
 3. Reactor Benchmarking
 
 ### Publications 
-1. R. Stewart, C. Pope and E. Ryan, "Fast spectrum reactor fuel assembly sensitivity analysis", Annals of Nuclear Energy, vol. 110, pp. 1091-1097, 2017.
+1. R. Stewart, and T. Palmer, “Metallic Fuel Design Space for Sodium Fast Reactors,” 	Transactions of the American Nuclear Society, vol. 119, 2018, pp. 1116-1119.
+2. D. LaBrier, R. Stewart, et al., “Design of an In-Pile Experimental Loop for Fast Test Reactor 	Conditions,” Transactions of the American Nuclear Society, vol. 119, 2018, pp. 1271-1274.
```
Comment (nickwhitman1993) [inline users/stewryan/index.md]: extra white space

Comment (ryanstwrt) [reply users/stewryan/index.md]: This is what I get from copying and pasting! It should be good now.

**Comment by nickwhitman1993** [comment]
Looks good to me. @aaronjamesreynolds :shipit:?

**Comment by aaronjamesreynolds** [comment]
Looks good to me! 👍

**Pull request merged**